### PR TITLE
Milestone: Mobile menu

### PR DESCRIPTION
**Pull request by davmba**
**### Mobile menu milestone: Portfolio**
**In this milestone, we:**

- Implement the mobile menu using the recently acquired JavaScript knowledge.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that involves JavaScript but doesn't have JavaScript linters set up.  I kindly suggest that you add [html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linters to your project and ensure all your linters including ESLint are green.

## ESLint AutoFix
- if you have a lot of JavaScript line errors, I kindly suggest you fix them by running `npx eslint . --fix`, but please get used to fixing the errors line by line also because it'll help you learn best practices.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -9,31 +9,32 @@
...
+              <li><a href="#portfolio" class="nav-link">Portfolio</a></li>
+              <li><hr></li>
+              <li><a href="#about-me" class="nav-link">About</a></li>
+              <li><hr></li>
+              <li><a href="#contact-us" class="nav-link"> Contact</a></li>
+              <li><hr></li>
               <li><i class="ri-mail-line"></i></li>
             </ul>
           </nav>
         </div>
```
Comment (mirouhml) [inline index.html:33]: - [ ] Please know that the mobile menu should cover all of the screen, kindly fix this:

image

File: index.html [hunk 2]
```diff
@@ -9,31 +9,32 @@
...
+          <h3>Dave Mba.</h3>
         </div>
         <div class="nav">
           <nav>
+            <i class="fa-solid fa-xmark xmark"></i>
             <ul>
-              <li><a href="">Portfolio</a></li>
-              <li><a href="">About</a></li>
-              <li><a href=""> Contact</a></li>
+              <li><a href="#portfolio" class="nav-link">Portfolio</a></li>
```
Comment (mirouhml) [inline index.html:24]: - [ ] Please know that the portfolio link should take the user to the `recent works` sections, kindly fix this. 😁

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @davmba, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team, 👋🏻👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥🔥🎉🎉


## Highlights ✨✨
- No Linters error ✅✅
- Great job on working on the requested changes ✅✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**